src/prefect/task_worker.py

```diff
@@ -362,7 +388,7 @@ async def __aenter__(self):
         self._started_at = pendulum.now()
         return self
 
-    async def __aexit__(self, *exc_info):
+    async def __aexit__(self, *exc_info: Any):
```

Comment:
You forgot the return annotation 😀 

```suggestion
    async def __aexit__(self, *exc_info: Any) -> Optional[bool]
```

Or just None or bool as applicable.

Reply:
sorry, maybe I'm missing your point but this function doesn't `return`

Reply:
Well, either it is a `NoReturn` (highly unlikely) or the default for a function without a return statement, `None`.